vector/src/main/java/im/vector/app/features/home/room/list/SpaceChildInfoItem.kt

```diff
@@ -63,7 +63,7 @@ abstract class SpaceChildInfoItem : VectorEpoxyModel<SpaceChildInfoItem.Holder>(
             it.performHapticFeedback(HapticFeedbackConstants.LONG_PRESS)
             itemLongClickListener?.onLongClick(it) ?: false
         }
-        holder.titleView.text = matrixItem.getBestName()
+        holder.titleView.text = matrixItem.displayName ?: holder.rootView.context.getString(R.string.unnamed_room)
```

Comment:
Either provide a stringProvider as an EpoxyAttribute, or maybe better, provide the titleText in the controller

Reply:
why use stringprovider when i have access to context?

Reply:
Binding items should compute the less thing as possible